mix.exs

```diff
@@ -48,19 +48,12 @@ defmodule EpochElixir.Mixfile do
       {:credo, "~> 0.9.3", only: [:dev, :test], runtime: false},
       {:dialyxir, "~> 0.5", only: [:dev, :test], runtime: false},
       {:sha3, [github: "szktty/erlang-sha3", ref: "dbdfd12", manager: :rebar]},
-      {:idna, [github: "aeternity/erlang-idna", ref: "24bf647", manager: :rebar, override: true]},
-      {:gen_state_machine, "~> 2.0.1"},
-      {:logger_file_backend, "~> 0.0.10"},
-      {:excoveralls, "~> 0.8.1", only: :test},
```

Comment:
I removed only gen_state_machine, another deps I just moved to the correct mix files, but I will return excoveralls because it is used not only in aecore

Reply:
@0legIv did you adjust to @cytadela8 recommendations?

Reply:
@cytadela8 Yes, but if the dependency for example is used in more than one app, should I move it to these apps and have duplications, or it is better to leave it in the general mix.exs?

Reply:
as @cytadela8 said "You can move dependencies that are used in many apps to corresponding apps"